nipype/interfaces/mrtrix3/utils.py

```diff
@@ -677,3 +677,86 @@ def _list_outputs(self):
         outputs = self.output_spec().get()
         outputs['out_file'] = op.abspath(self.inputs.out_file)
         return outputs
+
+
+class MRResizeInputSpec(MRTrix3BaseInputSpec):
+    in_file = File(
+        exists=True,
+        argstr='%s',
+        position=-2,
+        mandatory=True,
+        desc='input DWI image'
+    )
+    image_size = traits.Tuple(
+        (traits.Int, traits.Int, traits.Int),
+        argstr='-size %d,%d,%d',
+        mandatory=True,
+        desc='define the new image size for the output image. This should be '
+        'specified as a comma-separated list.',
+        xor=['voxel_size', 'scale_factor'],
+    )
+    voxel_size = traits.Tuple(
+        (traits.Float, traits.Float, traits.Float),
+        argstr='-voxel %d,%d,%d',
```

Comment:
If these can be floats, we need to use `%f` or `%g`:

```suggestion
        argstr='-voxel %g,%g,%g',
```